homeassistant/components/modbus/modbus.py

```diff
@@ -255,6 +256,16 @@ class ModbusHub:
     def __init__(self, hass: HomeAssistant, client_config: dict[str, Any]) -> None:
         """Initialize the Modbus hub."""
 
+        if CONF_CLOSE_COMM_ON_ERROR in client_config:
```

Comment:
Typically we also do logging so we do both.

Reply:
Done.